static/calculate_score.html

```diff
@@ -62,11 +62,16 @@
         <div class="col-12 col-md-8 mx-auto">
           <div class="row text-center">
             <div class="text-center col-12">
-              <h3>1: Input RS IDs and Alleles</h3>
```

Comment:
Remove/replace line 50 (that talks about 7 easy steps) since it's different now.